kafka-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/kafka/autoconfig/KafkaAutoConfigurationIntegrationTest.java

```diff
@@ -555,7 +564,9 @@ protected static class TestConfiguration {
 
         @Bean
         public Serializer eventSerializer() {
-            return XStreamSerializer.defaultSerializer();
+            return XStreamSerializer.builder()
```

Comment:
Just wondering why this change? Just so it won't log for using the default and/or because it's deprecated?

Reply:
Since I was changing serializers, I get rid of the deprecated method and migrated it to the 'new way'. Tests should still be the same as before, just a code smell less =)